python/tk_multi_workfiles/file_list/file_list_form.py

```diff
@@ -131,12 +141,55 @@ def __init__(
         self._item_delegate = FileListItemDelegate(self._ui.file_list_view)
         self._ui.file_list_view.setItemDelegate(self._item_delegate)
 
+        # Set up the filter menu.
+        self._filter_menu = FilterMenu(self, refresh_on_show=False)
+        # The list of fields that the menu will show.
+        self._filter_menu.set_accept_fields(
+            [
+                "{role}.is_local".format(role=FileModel.FILE_ITEM_ROLE),
+                "{role}.is_published".format(role=FileModel.FILE_ITEM_ROLE),
+                "{role}.name".format(role=FileModel.FILE_ITEM_ROLE),
+                "{role}.step".format(role=FileModel.FILE_ITEM_ROLE),
+                "{role}.asset".format(role=FileModel.FILE_ITEM_ROLE),
+                "{role}.asset_type".format(role=FileModel.FILE_ITEM_ROLE),
+                "{role}.modified_at".format(role=FileModel.FILE_ITEM_ROLE),
+                "{role}.path".format(role=FileModel.FILE_ITEM_ROLE),
+                "{role}.publish_path".format(role=FileModel.FILE_ITEM_ROLE),
+                "{role}.published_at".format(role=FileModel.FILE_ITEM_ROLE),
+                "{role}.published_by".format(role=FileModel.FILE_ITEM_ROLE),
+                "{role}.published_description".format(role=FileModel.FILE_ITEM_ROLE),
+                "{role}.task".format(role=FileModel.FILE_ITEM_ROLE),
+                "{role}.version".format(role=FileModel.FILE_ITEM_ROLE),
+            ]
+        )
```

Comment:
I think one alternative to rewriting this could be to have a list of fields such as `["is_local", "is_published", ...]` and then iterate them like

```py
self._filter_menu.set_accept_fields(
    ["{role}.{field}".format(role=FileModel.FILE_ITEM_ROLE, field=field) for field in fields]
)
```

Just a personal suggestion, not a blocker for moving forward.